libs/tasks/data-access/src/lib/+state/tasks/tasks.reducer.ts

```diff
@@ -2,17 +2,17 @@ import { createReducer, on, Action } from '@ngrx/store';
 import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
 
 import * as TasksActions from './tasks.actions';
-import { Task } from '../../../../../resources/models/task';
+import { Task } from '../../resources/models/task';
 
 export const TASKS_FEATURE_KEY = 'tasks';
 
 export interface TasksState extends EntityState<Task> {
   tasks: Task[];
-  selectedId?: string | number; // which Tasks record has been selected
+  selectedId: string | null; // which Tasks record has been selected
```

Comment:
I think you don't need null property. Just set an empty string

Reply:
zmieniłem w końcu na number więc zostawiłem też null